src/applications/disability-benefits/all-claims/utils/submit.js

```diff
@@ -255,10 +255,10 @@ export const stringifyRelatedDisabilities = formData => {
       facility,
     ),
   );
-  if (newVAFacilities.length) {
-    clonedData.vaTreatmentFacilities = newVAFacilities;
-  } else {
+  if (newVAFacilities.length === 0 || clonedData.syncModern0781Flow === false) {
```

Comment:
I am sure I am just reading this wrong or not understanding the context further up but just incase...

If there are `>0 newVAFacilities`, and `clonedData.syncModern0781Flow = false` This would delete the facilites unintentionally?

IE I have 5 facilites entered (first logic is false) and am not in the new flow (second logic is true, so with the or, it is true overall), so the facilites get deleted?  

That doesnt seem right? Or am I trippin'

Reply:
Reviewing, I believe you're correct. Going to revisit and revise as needed.

Reply:
Reverted to original logic in [329b679](https://github.com/department-of-veterans-affairs/vets-website/pull/35022/commits/329b679f58aa72fc5bd556f16eb46be188d2c307) as evaluating `syncModern0781Flow ` was unnecessary if `newVAFacilities ` was empty.